BUILDING.md

```diff
@@ -476,6 +476,14 @@ Prerequisites:
   If not installed in the default location, it needs to be manually added
   to `PATH`. A build with the `openssl-no-asm` option does not need this, nor
   does a build targeting ARM64 Windows.
+  
+Note: To install all prerequisites you can install node-gyp with npm using the following command
+
+```console
+> npm install -g node-gyp
```

Comment:
`node-gyp` is not required to build Node.js. 

While I have not tried `windows-build-tools` I don't think it includes `git` or `nasm`,

Reply:
Hi, thanks for the feedback. Please pardon my vocabulary. I was unable to build node until I installed node-gyp. I initially attempted to install Windows-build-tools alone but it failed. Maybe it failed because of some missing dependency but after I installed node-gyp node built successfully.

Reply:
Yes, you are correct that node-gyp does not include git or nasm. I installed those separately, especially the Net Assembler, because I had git already installed. What about this optional requirement for windows 10 - can this be were the failure to build came from?

* **Optional** Requirements for compiling for Windows 10 on ARM (ARM64):
    * ARM64 Windows build machine
        * Due to a GYP limitation, this is required to run compiled code
          generation tools (like V8's builtins and mksnapshot tools)